### ISSUE TITLE
u3d doesn't recognize Unity 2018.4.32

### ISSUE THREAD

**Issue by Duobix** [DragonBox/u3d#413]
### Issue Checklist

- [ x] I'm using the latest u3d version
- [ x] I ran `u3d --help`
- [ x] I read the [README](https://github.com/DragonBox/u3d/blob/master/README.md)
- [ x] I made sure that a similar [issue](https://github.com/DragonBox/u3d/issues) doesn't exit

### Issue Description

[DESCRIBE YOUR ISSUE HERE]
I have installed Unity 2018.4.32 by the Unity Hub, then wanted to run a project with u3d, but it seems it's not recognized by u3d at all. By listing all the unity versions, every single one shows up, but not 2018.4.32 installed in Unity Hub. The version is in the same folder as other versions that are recognized by u3d, and also 2018.4.32 is listed as available for download on u3d.

**Comment by crowder-pw** [comment]
Yes, wondering if `u3d` simply doesn't detect hub-installed versions?

**Comment by mattmaniak** [comment]
@crowder-pw it does in my case but sometimes says "no" without a reason.

**Comment by lacostej** [comment]
Hello, I missed this issue. On which platforms are you running u3d? On mac, when the finder indexing is running, Unity should find the versions installed by the hub. On the other platforms, we lack the code to search for the editors that hub installed.

We could for example parse `~/Library/Application\ Support/UnityHub/editors.json` on Mac. If you let me know where this file is on Windows and Linux we could autodetect those installs as well.

Thanks
